DPPP/DPInfo.cc

```diff
@@ -56,11 +56,12 @@ namespace DP3 {
         itsPhaseCenterIsOriginal (true)
     {}
 
-    void DPInfo::init (uint ncorr, uint nchan,
+    void DPInfo::init (uint ncorr, uint startChan, uint nchan,
                        uint ntime, double startTime, double timeInterval,
                        const string& msName, const string& antennaSet)
     {
       itsNCorr        = ncorr;
+      itsStartChan = startChan;
```

Comment:
Please use consistent indentation in files.

Reply:
Are you referring to the fact that the equal signs don't align? I'll fix it, but I find this kind of over-spacing really annoying and a waste of time, it just brings work to keep it like this and is really not more readable. In fact, sometimes the equal sign in DP3 is so far from the variable that you have to place your finger on the screen to see which is which. Proper tooling will already highlight and colour the code, which makes it as readable as possible.